src/Forms/GridField/GridFieldDataColumns.php

```diff
@@ -31,6 +34,8 @@ class GridFieldDataColumns extends AbstractGridFieldComponent implements GridFie
      */
     protected $displayFields = [];
 
+    private bool $escapeFields = true;
```

Comment:
```suggestion
    private bool $doEscapeFields = true;
```

Because "escapeFields" is suffixed with "fields" I expect it to be a FieldList, or an array, not a bools. The "do" prefix is a fairly standard to make it clear a variable is a bool

Also update the getters/setters and references in this PR

Reply:
Done